package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-write-good-comments",
-  "version": "0.1.4",
+  "version": "0.1.5",
```

Comment:
looks good to me, but I could you change so that the version shows that there is a new feature?

```suggestion
  "version": "0.2.0",
```

Reply:
Done!